reverseCase/reverseCase.js

```diff
@@ -3,5 +3,20 @@
  * Example: 'Hello World' -> 'hELLO wORLD'
  * Assume that each string will contain only spaces and letters.
  */
+const newString = [];
 
+const lowerUpper = (string) => {
+  for (let i = 0; i < string.length; i++) {
+    const character = string[i];
+    if (character === character.toLowerCase()) {
```

Comment:
Awesome. I love the readability of your code. It is nice and clean, and it flows really well. I would be glad to receive this file if I was working on a team with you! Keep it up.